index.html

```diff
@@ -4,43 +4,193 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio Main Page</title>
+    <title>Grid and Flexbox</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container">
-        <header class="box header">
-            <nav>
-                <span><a href="index.html">My Logo</a></span>
-                <span class ="hamburger"><img src="img/Enabled.png"></span>
-            </nav>
-        </header>
-        <section class="box content">
-            <h1 class="title">
-            I’m Mariane 
-            Glad to see 
-            you!
-            </h1>
-            <p class="paragraph">
-                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
-                   If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            
-            <div class="connect">
+    <div class="grid-container">
+        <div class="grid-item flex-container">
+            <div class ="flex-item flex-item-1"><img class="deneme" src="../img/Project-1.png"></div>
+            <div class ="flex-item flex-item-2"><h1 class="project-content-2">Tonic</h1></div>
```

Comment:
- [ ] [REQUIRED] For SEO reasons and semantics, you should not have more than one `h1` element on your page. I presume you have `h1` already in your headline section. Yet you have `h1` again more than once on the page. Please ensure your replace them with `h2`, or `h3`.